components/script/dom/htmlformelement.rs

```diff
@@ -467,12 +467,21 @@ impl HTMLFormElement {
     fn static_validation(&self) -> Result<(), Vec<FormSubmittableElement>> {
         let node = self.upcast::<Node>();
         for child in node.traverse_preorder() {
-            match child.downcast::<Element>() {
-                Some(el) if !el.disabled_state() => (match el.as_maybe_validatable() {
-                    Some(ele) => (if ele.is_instance_validatable() {}),
-                    None => {}
-                }),
-                _ => continue,
+            if let Some(el) = child.downcast::<Element>() {
+                if el.disabled_state() {
+                    continue;
+                }
+                let validatable = match el.as_maybe_validatable() {
+                    Some(v) => v,
+                    None => continue,
+                };
+                if !validatable.is_instance_validatable() {
+                    continue;
+                }
+                let mut validate_flags: ValidationFlags = ValidationFlags::empty();
+                // Need match and insert ValidationFlags later
+                validatable.validate(validate_flags);
```

Comment:
I would go ahead and call `validatable.validate(ValidationFlags::empty())` for the time being.